QuestionSets/PythonLanguage/BasicsAndArrays/Manan_Garg/sol_3manan.py

```diff
@@ -0,0 +1,3 @@
+import numpy as np
+A = np.arange(0,6,1).reshape(3,2)
```

Comment:
@Manan1504 this will print `[[0, 1], [2, 3], [4, 5]]` not `[[1, 2], [2, 3], [4, 5]]`. You are close, just one change should do it!

Reply:
But the question set tells to print [[0, 1], [2, 3], [4, 5]]

Reply:
do I need to make a pull request now?

Reply:
nm